requirements.txt

```diff
@@ -0,0 +1,18 @@
+aiosignal==1.3.1
```

Comment:
download black, format your project, then check the cases with ' and ", where it can occur the issue